src/Keboola/DbExtractor/Test/ExtractorTest.php

```diff
@@ -8,14 +8,17 @@
 
 class ExtractorTest extends \PHPUnit_Framework_TestCase
 {
+    /**
+     * @var string
+     */
     protected $dataDir = ROOT_PATH . "/tests/data";
 
     /**
      * @param $driver
      * @param string $format (yaml or json)
-     * @return mixed
+     * @return array|mixed
```

Comment:
pretty sure it's not mixed due to typehint, also $driver is missing type in docblock

Ideally format would be class constant `self::CONFIG_FORMAT`, making it's obvious what available options are.